tools/engine_tool/lib/src/build_plan.dart

```diff
@@ -84,14 +90,57 @@ final class BuildPlan {
     required this.useRbe,
     required this.useLto,
     required this.concurrency,
-  }) {
+    required Iterable<String> extraGnArgs,
+  }) : extraGnArgs = List.unmodifiable(extraGnArgs) {
     if (!useRbe && strategy == BuildStrategy.remote) {
       throw FatalError(
         'Cannot use remote builds without RBE enabled.\n\n$_rbeInstructions',
       );
     }
   }
 
+  /// Arguments that cannot be provided to [BuildPlan.extraGnArgs].
+  ///
+  /// Instead, provide them explicitly as other [BuildPlan] arguments.
+  @visibleForTesting
+  static const reservedGnArgs = {
```

Comment:
do we have a map of "bad flag" -> "et flag"? that might help whoever is running `et --extraflags`

Reply:
In this particular case, the arguments are identical.

I can clarify that in the error message and leave a breadcrumb that if we expand this list to ensure it's clear to the user.

Reply:
Done!